### DISCUSSION TITLE
Window management

### DISCUSSION THREAD

**Discussion by hodgesd** [ttscoff/bunch-docs#6] [Q&A] [answered]
Is it possible to perform simple window management win opening apps?  Perhaps sending them to the left or right half of the screen?

Thanks.

**Comment by ttscoff** [comment, marked as answer]
I've chosen, at least for now, to leave window management up to other apps. Bunch easily integrates with apps like Moom or BetterTouchTool for sizing and positioning windows. See the [Integration section of the docs](https://brettterpstra.com/bunch-beta/docs/integration/) for more info.
